media-plugins/mythplugins/mythplugins-30.0.ebuild

```diff
@@ -0,0 +1,167 @@
+		>=media-sound/lame-3.93.1
+	)
+	ieee1394? (
+		>=media-libs/libiec61883-1.0.0
+		>=sys-libs/libavc1394-0.5.3
+		>=sys-libs/libraw1394-1.2.0
+	)
+	libass? ( >=media-libs/libass-0.9.11:= )
+	mytharchive? (
+		app-cdr/dvd+rw-tools
+		dev-python/pillow
+		media-video/dvdauthor
+		media-video/mjpegtools[png]
+		media-video/transcode
+		virtual/cdrtools
+	)
+	mythbrowser? ( dev-qt/qtwebkit:5 )
+	mythgallery? (
+		media-libs/tiff:0
+		opengl? ( virtual/opengl:= )
+		exif? ( >media-libs/libexif-0.6.9:= )
+		raw? ( media-gfx/dcraw )
+	)
+	mythgame? ( !syslibs/zlib[minizip] )
+	mythmusic? (
+		>=media-libs/flac-1.1.2
+		media-libs/libogg
+		>=media-libs/libvorbis-1.0
+		>=media-libs/taglib-1.6
+		>=media-sound/lame-3.93.1
+		fftw? ( sci-libs/fftw:3.0= )
+		opengl? ( virtual/opengl )
+		cdda? (
+			dev-libs/libcdio:=
+			cdr? ( virtual/cdrtools )
+		)
+	)
+	mythnetvision? (
+		${PYTHON_DEPS}
+		dev-python/pycurl
+		dev-python/lxml
+		dev-python/mysql-python
+		dev-python/oauth
+	)
+	mythweather? (
+		dev-perl/Date-Manip
+		dev-perl/XML-Simple
+		dev-perl/XML-XPath
+		dev-perl/DateTime
+		dev-perl/Image-Size
+		dev-perl/DateTime-Format-ISO8601
+		dev-perl/SOAP-Lite
+		dev-perl/JSON
+	)
+	theora? ( media-libs/libtheora )
+	xml? ( >=dev-libs/libxml2-2.6.0:= )
+	xvid? ( >=media-libs/xvid-1.1.0 )
+"
+RDEPEND="${DEPEND}
+	!<media-tv/mythtv-${PV}[alsa=,hls=,ieee1394=,libass=,python,theora=,vorbis=,xml=,xvid=]
+	!media-plugins/mytharchive
+	!media-plugins/mythbrowser
+	!media-plugins/mythgallery
+	!media-plugins/mythgame
+	!media-plugins/mythmovies
+	!media-plugins/mythmusic
+	!media-plugins/mythnetvision
+	!media-plugins/mythnews
+	!media-plugins/mythweather
+	!media-plugins/mythzmserver
+	!media-plugins/mythzoneminder
+"
+REQUIRED_USE="
+	mythmusic? ( vorbis )
+	mythnetvision? ( ${PYTHON_REQUIRED_USE} )
+	mythnews? ( mythbrowser )
+"
+
+# mythtv and mythplugins are separate builds in the github MythTV repository
+S="${WORKDIR}/${P}/mythplugins"
+
+DOC_CONTENTS="
+All plugins are installed by default. Disable unneeded plugins individually with USE flags:
+-mytharchive -mythbrowser -mythgallery -mythgame -mythmusic \
+-mythnetvision -mythnews -mythweather -mythzmserver -mythzoneminder
+"
+
+src_prepare() {
+	default
+	sed -i '1i#define OF(x) x' mythgame/mythgame/external/ioapi.h
+}
+
+src_configure() {
+	econf \
+		--prefix=/usr \
+		--python=${EPYTHON} \
+		--extra-ldflags="${LDFLAGS}" \
+		$(use_enable cdda cdio) \
+		$(use_enable exif) \
+		$(use_enable exif new-exif) \
+		$(use_enable fftw) \
+		$(use_enable opengl) \
+		$(use_enable raw dcraw) \
+		$(use_enable mytharchive) \
+		$(use_enable mythbrowser) \
+		$(use_enable mythgallery) \
+		$(use_enable mythgame) \
+		$(use_enable mythmusic) \
+		$(use_enable mythnetvision) \
+		$(use_enable mythnews) \
+		$(use_enable mythweather) \
+		$(use_enable mythzmserver) \
+		$(use_enable mythzoneminder)
+}
+
+src_install() {
+	emake INSTALL_ROOT="${D}" install || die "make install failed"
```

Comment:
`emake` doesnt need to die, its a built-in helper function :)

Reply:
OK fixed.